build.sbt

```diff
@@ -1,71 +1,74 @@
 lazy val common = Seq(
-  scalaVersion := "2.12.6",
+  //  scalaVersion := "2.12.6",
+  scalaVersion := "2.13.6",
   crossScalaVersions := Seq("2.11.12", "2.12.6"),
```

Comment:
`crossScalaVersions` should be probably changed to `Seq("2.11.12", "2.12.12", "2.13.6")`

Reply:
Done